src/errors.d

```diff
@@ -18,6 +18,8 @@ import ddmd.globals;
 import ddmd.root.outbuffer;
 import ddmd.root.rmem;
 
+//debug = printGaggedErrors;
```

Comment:
Since I've used the debug feature at least 100 times.
So, if I'll use the debug feature more 100 times in the future, I would need to modify 200 lines to uncomment code.
After this change, I'll need to modify only 100 lines. That's enough valuable.